packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -84,6 +84,19 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
           },
           '&:focus': {
             color: semanticColors.link
+          },
+          '&:focus::after': {
+            content: '" "',
+            position: 'absolute',
+            left: '0',
+            top: '0',
+            right: '0',
+            bottom: '0',
+            boxShadow: '0 0 0 1px rgba(0, 0, 0, 0.8) inset'
+          },
+          '&image': {
```

Comment:
This rule doesn't make sense to me.

Reply:
If the intent is to special case images, the selector would have been `& img` as the element is an "img" tag. But... that would be a bad practice as well.

In this case you're creating a selector, where "&" represents the generated classname, such as "foo-123". So your final selector would be "foo-123image" which seems quite wrong. :)

What is the intent? I don't think you should set all images within the link to inline-block...

Reply:
Hmm maybe I misunderstood the image/try3 combo from https://codepen.io/dzearing/pen/ZmewZB. I thought we had to set any images that are wrapped in links to inline-block and the lineheight to 0 in order for them to fit inside the focus rectangle around the whole link...